render/render.js

```diff
@@ -99,8 +99,8 @@ module.exports = function($window) {
 		ns = getNameSpace(vnode) || ns
 
 		var element = ns ?
-			is ? $doc.createElementNS(ns, tag, {is: is}) : $doc.createElementNS(ns, tag) :
-			is ? $doc.createElement(tag, {is: is}) : $doc.createElement(tag)
+			is !== undefined ? $doc.createElementNS(ns, tag, {is: is}) : $doc.createElementNS(ns, tag) :
```

Comment:
`is != null` is preferred here.

Reply:
I can adjust if the basis of this PR fit specs.